DEPS

```diff
@@ -614,7 +614,7 @@ deps = {
   Var('dart_git') + '/external/github.com/google/file.dart.git' + '@' + 'b2e31cb6ef40b223701dbfa0b907fe58468484d7', # 6.1.4
 
   'src/third_party/pkg/flutter_packages':
-  Var('github_git') + '/flutter/packages.git' + '@' + '26990a2f75ab2028c3c77ffc869db11d6d866d18', # various
+  Var('github_git') + '/flutter/packages.git' + '@' + '25454e63851fe7933f04d025606e68c1eac4fe0f', # various
```

Comment:
Not sure the process here, but FYI there's a new ToT at https://github.com/flutter/packages/commit/e1ab965985d4e8ab5fdcf1b9989cf1c22d770205